frontend/src/pages/WhatIsHealthEquity/WhatIsHealthEquityPage.tsx

```diff
@@ -318,59 +377,78 @@ function WhatIsHealthEquityPage() {
             <br />
           </Grid>
           <Grid container className={styles.JoinTheEffortItemContainer}>
+            <Hidden smDown>
+              <Grid
+                item
+                xs={5}
+                className={styles.JoinTheEffortImgContainer}
+                style={{ backgroundColor: "#275141" }}
+              >
+                <img
+                  src="img/Dots_1@2x 4.png"
+                  alt="Decorative dots"
+                  className={styles.JoinTheEffortImg}
+                />
+              </Grid>
+            </Hidden>
             <Grid
               item
-              xs={5}
-              className={styles.JoinTheEffortImgContainer}
-              style={{ backgroundColor: "#275141" }}
+              sm={12}
+              md={7}
```

Comment:
was md={5} wonky?

Reply:
GitHub review tool isn't lining up properly... it was originally md={7} and I left it as such. See line 333 in original version